src/JustSaying/Messaging/Middleware/Backoff/BackoffMiddlewareBuilderExtensions.cs

```diff
@@ -13,14 +13,13 @@ public static class BackoffMiddlewareBuilderExtensions
         /// a <see cref="BackoffMiddleware"/> and add it to the pipeline.
         /// </summary>
         /// <param name="builder">The <see cref="HandlerMiddlewareBuilder"/> to add the middleware to.</param>
+        /// <param name="backoffStrategy">The <see cref="IMessageBackoffStrategy"/> to use to determine message visibility timeouts.</param>
         /// <returns>The current <see cref="HandlerMiddlewareBuilder"/>.</returns>
         /// <exception cref="ArgumentNullException">When the <see cref="HandlerMiddlewareBuilder"/> is null.</exception>
-        public static HandlerMiddlewareBuilder UseBackoff(this HandlerMiddlewareBuilder builder)
+        public static HandlerMiddlewareBuilder UseBackoff(this HandlerMiddlewareBuilder builder, IMessageBackoffStrategy backoffStrategy)
```

Comment:
Nit: null check the back-off strategy.

Reply:
Ah thanks, missed this! 👍

Reply:
Did you forget to push the fix?

Reply:
🤦‍♂️ I pushed it to my other branch instead of here hah. Done now, sorry about that!